teledata/models.py

```diff
@@ -4,13 +4,60 @@
 import logging
 
 from django.db import models
-from django.db.models import Q, When, Case, Value
-from django.db.models.expressions import RawSQL
-from django_mysql.models import QuerySet
-from django_mysql.models import QuerySetMixin
+from django.db.models import F, When, Case, Value, Expression
+from django_mysql.models import QuerySet, QuerySetMixin
 
 logger = logging.getLogger(__name__)
 
+class MatchAgainst(Expression):
```

Comment:
A comment explaining the purpose of this class/general description of what it's doing would be helpful for future reference

Reply:
Comment added.